deployment/Makefile

```diff
@@ -68,6 +68,11 @@ collectstatic:
 	#We need to run collect static in the same context as the running
 	# uwsgi container it seems so I use docker exec here
 	@docker exec -t -i $(PROJECT_ID)_uwsgi_1 python manage.py collectstatic --noinput
+	#
+	# RM : this is needed to copy some library over
+	# I don't know a better way to do this
+	sudo cp ../django_project/feti/static/feti/js/ZeroClipboard.swf static/js/
```

Comment:
@lucernae, could you maybe explain what you are doing here in a bit of detail. From the face of it, I would think that you could achieve something similar with a mount in docker-compose?